### ISSUE TITLE
Error on bare app

### ISSUE THREAD

**Issue by emac3** [expo/match-media#14]
I get the following error when using it on iOS:

`ExceptionsManager.js:44 Error: The Expo SDK requires Expo to run. It appears the native Expo modules are unavailable and this code is not running on Expo. Visit https://docs.expo.io to learn more about developing an Expo project.`

Installing `expo-screen-orientation` and the amending the following import in `node_modules/@expo/match-media/build/polyfill.native.js` apparently solves the issue:

~~`import { ScreenOrientation } from "expo";`~~
`import * as ScreenOrientation from 'expo-screen-orientation';`

`expo diagnostics`:
```
  Expo CLI 3.20.1 environment info:
    System:
      OS: macOS Mojave 10.14.6
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 12.16.3 - ~/.nvm/versions/node/v12.16.3/bin/node
      Yarn: 1.22.4 - /usr/local/bin/yarn
      npm: 6.14.4 - ~/.nvm/versions/node/v12.16.3/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.6 AI-192.7142.36.36.6392135
      Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
    npmPackages:
      expo: ~37.0.3 => 37.0.7
      react-native: ~0.61.5 => 0.61.5
```

**Comment by thegeekybaniya** [comment]
Tried this, got another error

![image](https://user-images.githubusercontent.com/22905237/85168240-b0842200-b287-11ea-9426-6e38b00956d1.png)

expo diagnostics:
![image](https://user-images.githubusercontent.com/22905237/85168297-c8f43c80-b287-11ea-93cc-d491acefb566.png)

**Comment by EvanBacon** [comment]
Fixed in 0.0.0-beta.2
